spatialos-sdk/src/worker/schema.rs

```diff
@@ -0,0 +1,662 @@
+use crate::worker::{
+    component::{Component, ComponentId, ComponentUpdate},
+    internal::Sealed,
+    EntityId,
+};
+use spatialos_sdk_sys::worker::*;
+use std::{collections::BTreeMap, marker::PhantomData};
+
+pub type FieldId = u32;
+
+#[derive(Debug)]
+pub struct SchemaComponentUpdate<'a> {
+    pub(crate) internal: *mut Schema_ComponentUpdate,
+    fields: SchemaObject<'a>,
```

Comment:
Is there a reason that this one is special and we inline the `SchemaObject`? Whereas the other `Schema*` types we don't do a similar thing.

Also this is missing events :wink:

Reply:
The answer to both of those is "Work In Progress" 🤷‍♀️